packages/app/src/app/pages/NewDashboard/Components/Filters/ViewOptions/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
```suggestion
import React, { ComponentType, FunctionComponent } from 'react';
```

Reply:
I don't really see the sue in adding that, it has never helped me really :/